src/Microsoft.AspNet.Mvc.Core/ConsumesAttribute.cs

```diff
@@ -53,13 +64,12 @@ public void OnResourceExecuting(ResourceExecutingContext context)
             // Ignore all other filters. This is to ensure we have a overriding behavior.
             if (IsApplicable(context.ActionDescriptor))
             {
-                MediaTypeHeaderValue requestContentType = null;
-                MediaTypeHeaderValue.TryParse(context.HttpContext.Request.ContentType, out requestContentType);
+                var requestContentType = context.HttpContext.Request.ContentType;
 
                 // Confirm the request's content type is more specific than a media type this action supports e.g. OK
                 // if client sent "text/plain" data and this action supports "text/*".
                 if (requestContentType != null &&
```

Comment:
What I meant is that user in the server code will not do request.Headers["Content-Type"] = "", that the content type typically comes in from the request from the client and that it's not normal that a client sends Content-Type: (which i don't even know if would result in Content-Type: "" or Content-Type: null or our current stack